### PR TITLE
fix: change occasional usage of kebab case to snake case

### PR DESCRIPTION
**Pull request by Tabrizian**
There are some occasional usage of kebab case in the cli options

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfctl/263)

### CODE REVIEW

File: pkg/apis/apps/group.go [hunk 1]
```diff
@@ -97,10 +97,10 @@ const (
 	APPNAME               CliOption = "appname"
 	DATA                  CliOption = "Data"
 	ZONE                  CliOption = "zone"
-	USE_BASIC_AUTH        CliOption = "use_basic_auth"
-	USE_ISTIO             CliOption = "use_istio"
-	DELETE_STORAGE        CliOption = "delete_storage"
-	DISABLE_USAGE_REPORT  CliOption = "disable_usage_report"
+	USE_BASIC_AUTH        CliOption = "use-basic-auth"
```
Comment (jlewi) [inline pkg/apis/apps/group.go]: Could we remove the options USE_BASIC_AUTH and USE_ISTIO? I don't think these options are used anymore. I think those options predated the move to KFDef.

Comment (Tabrizian) [reply pkg/apis/apps/group.go]: @jlewi Done!

**Comment by jlewi** [comment]
Do you know why the tests are failing?

**Comment by Tabrizian** [comment]
@jlewi Not sure about why it fails exactly. 

http://testing-argo.kubeflow.org/workflows/kubeflow-test-infra/kubeflow-kfctl-presubmit-kfctl-go-iap-263-2630103-4805-8157?tab=workflow&nodeId=kubeflow-kfctl-presubmit-kfctl-go-iap-263-2630103-4805-8157-1236571111

Here is the test failing:
```
kubeflow-kfctl-presubmit-kfctl-go-iap-263-2630103-4805-8157.onExit.kfctl-delete-wrong-host

kubeflow-kfctl-presubmit-kfctl-go-iap-263-2630103-4805-8157.onExit
```

**Comment by jlewi** [comment]
@Tabrizian How did you try to access the logs? What was the stackdriver command you ran to fetch the logs?

I verified you are a member of the ci-viewers google group.

```
gam print group-members group ci-viewer@kubeflow.org
Getting members for ci-viewer@kubeflow.org (1/1)
group,id,email,role,type,status
ci-viewer@kubeflow.org, ...,iman.tabrizian@gmail.com,MEMBER,USER,
```

I also checked the IAM policy for kubeflow-ci.
[kubeflow.ci.policy.txt](https://github.com/kubeflow/kfctl/files/4355528/kubeflow.ci.policy.txt)

It looks like ci-viewer has loggging viewer and container viewer permissions which I believe should enable you to get the logs.

**Comment by Tabrizian** [comment]
@jlewi  Sorry, you're right I have access to the logs. But I can't figure out the command that I should use for the logs. It seems like there are multiple kubernetes clusters. Should I look for a specific Pod with a name similar to the one displayed in the Argo UI? 

At first, I meant the logs didn't show in the Argo UI.

**Comment by jlewi** [comment]
@Tabrizian you can follow these instructions to get the id of the pod that failed and then fetch the logs for the failed pod
https://github.com/kubeflow/testing#no-logs-in-argo-ui-for-step-or-pod-id-missing-in-argo-logs

Note: Make sure when using gcloud that you set freshness to be sufficiently long to include when your pod actually ran.

**Comment by PatrickXYS** [comment]
@Tabrizian Is this PR still necessary? Otherwise, I'll go ahead to close the PR

**Comment by Tabrizian** [comment]
@PatrickXYS I think it is necessary but it is up to you to merge it or not.

**Comment by jlewi** [comment]
Looks like tests are passing and this is a nice cleanup PR. So lets get it merged. I don't think I can approve this since I'm no longer an OWNER

@PatrickXYS Can you approve?

/lgtm

**Comment by PatrickXYS** [comment]
I don't think it's passing the test

/retest

**Comment by Tabrizian** [comment]
It looks like a rebase issue.

**Comment by PatrickXYS** [comment]
@Tabrizian Right, can you help rebase?

**Comment by Tabrizian** [comment]
@PatrickXYS  rebased

**Pull request merged**